client/my-sites/post-share/index.jsx

```diff
@@ -133,7 +123,7 @@ class PostShare extends Component {
 	};
 
 	sharePost = () => {
-		this.props.sharePost( this.props.siteId, this.props.post.ID, this.state.skipped, this.state.message );
+		this.props.sharePost( this.props.siteId, this.props.post.ID, this.state.message, this.state.enabledConnectionIds );
```

Comment:
just fyi, there is a `postId` property. You could use this one instead of `post.ID`.